shared/logger.js

```diff
@@ -0,0 +1,7 @@
+export default (tag) => ({
+  log: console.log.bind(console, tag),
+  debug: console.debug.bind(console, tag),
+  error: console.error.bind(console, tag),
+  info: console.debug.bind(console, tag),
+  warn: console.debug.bind(console, tag)
+})
```

Comment:
Why was this file created? And it doesn't seem to be used anywhere...? If not used, and no reason for it, plz delete

Reply:
It was created to avoid repeating the `[pubsub]` tag in every console call and to remove the following block in `backend/pubsub.js`:
```js
const log = console.log.bind(console, tag)
log.bold = (...args) => console.log(bold(tag, ...args))
log.debug = console.debug.bind(console, tag)
log.error = (...args) => console.error(bold.red(tag, ...args))
```
but it also needs bold and color support, so it's not ready for use yet